sidebars.js

```diff
@@ -783,6 +783,7 @@ module.exports = {
             items: [
               'mobile-apps/features/device-vitals',
```

Comment:
Although out of scope for this PR, the device-vitals.md file should be moved to the mobile-app-diagnostics folder like the other files in same structure.